github-pullrequest-plugin/src/main/resources/org/jenkinsci/plugins/github/pullrequest/events/impl/GitHubPRLabelAddedEvent/help.html

```diff
@@ -1,4 +1,5 @@
 <div>
-    Trigger when label was added to PR. When multiple label expected,
-    then job triggered only when all labels will appear in summary (in comparison to previous state).
+    Triggers a build when the label or labels specified are added to a
+    pull request. Note that with multiple labels, all of the labels must be
+    present to cause the trigger to fire.
```

Comment:
Misses the great part that triggering happens only in summary. Imagine: you requested in event 3 labels. On first check(state save) 1 label was added, on second check - 2 label was added. And real triggering happens `only when all labels will appear in summary (in comparison to previous state).`